quickstarts/microsoft.web/web-app-asp-app-on-asev3-create/modules/subnet.bicep

```diff
@@ -0,0 +1,23 @@
+@description('Required. The Virtual Network (vNet) Name.')
```

Comment:
remove defaultValues from the parameters since you prompt for them in main.bicep

Reply:
done.